docs/admin/reference/kubernetes/docker-jans-casa.md

```diff
@@ -83,7 +83,7 @@ The following environment variables are supported by the container:
 - `CN_GOOGLE_SECRET_NAME_PREFIX`: Prefix for Janssen secret in Google Secret Manager. Defaults to `jans`. If left `jans-secret` secret will be created.
 - `CN_GOOGLE_SECRET_MANAGER_PASSPHRASE`: Passphrase for Janssen secret in Google Secret Manager. This is recommended to be changed and defaults to `secret`.
 - `GLUU_CASA_APP_LOGGERS`: Custom logging configuration in JSON-string format with hash type (see [Configure app loggers](#configure-app-loggers) section for details).
-- `GLUU_CASA_ADMIN_LOCK_FILE`: Path to lock file to enable/disable administration feature (default to `/opt/jans/jetty/casa/resources/.administrable`). If file is not exist, the feature is disabled.
+- `GLUU_CASA_ADMIN_LOCK_FILE`: Path to lock file to enable/disable administration feature (default to `/opt/jans/jetty/jans-casa/.administrable`). If file is not exist, the feature is disabled.
```

Comment:
Path is `/opt/jans/jetty/jans-casa/resources/.administrable`

Reply:
The name would be `CN_CASA_ADMIN_LOCK_FILE` instead of `GLUU_CASA_ADMIN_LOCK_FILE`

Reply:
> /opt/jans/jetty/jans-casa/

In the 1.0.20 nightly, it is under `jans-casa` and not under `jans-casa/resources` :point_down: 

![image](https://github.com/JanssenProject/jans/assets/343411/1029a609-485f-4d83-ba78-9c0bc5f0904c)

Reply:
> The name would be `CN_CASA_ADMIN_LOCK_FILE` instead of `GLUU_CASA_ADMIN_LOCK_FILE`

@iromli I have fixed this with new commit.